data/README.md

```diff
@@ -0,0 +1,9 @@
+# Updating [git-platinum](https://github.com/radicle-dev/git-platinum)
+Just `cd` into the repo root, then run `scripts/update-git-platinum.sh`.
+This will update the tarball and it's necessary to commit the change. We provide a template below so that we can easily identify changes to `git-platinum`. Please fill in the details that follow a comment (`#`):
```

Comment:
Not to be a pain (but being a pain nonetheless): can we format the column size to 80? :)

Reply:
No worries, good point.